src/debug/Stream.h

```diff
@@ -37,10 +37,11 @@
 #define MAX_DEBUG_SECTIONS 100
 
 /* defined names for Debug Levels */
-#define DBG_CRITICAL    0   /**< critical messages always shown when they occur */
-#define DBG_IMPORTANT   1   /**< important messages always shown when their section is being checked */
-/* levels 2-8 are still being discussed amongst the developers */
-#define DBG_DATA    9   /**< output is a large data dump only necessary for advanced debugging */
+#define DBG_CRITICAL   0 /**< critical messages always shown when they occur */
+#define DBG_IMPORTANT  1 /**< important messages always shown when their section is being checked */
+#define DBG_PROTOCOL   2 /**< show flow of protocol messages. Excluding message content/payload which is in DBG_DATA. */
```

Comment:
For clarity and consistency sake:

```suggestion
#define DBG_PROTOCOL   2 /**< protocol messages excluding content/payload which is DBG_DATA */
```

or

```suggestion
#define DBG_PROTOCOL   2 /**< protocol message header/metadata (message content/payload is DBG_DATA) */
```